cpp/src/phonenumbers/base/memory/singleton_boost.h

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2013 The Libphonenumber Authors
```

Comment:
given that this is a new file, we should probably update this to 2020

Reply:
done (but code itself is older)